<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="12456" started-at="2022-07-30T08:51:33 IST" finished-at="2022-07-30T08:51:46 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="12456" started-at="2022-07-30T08:51:33 IST" finished-at="2022-07-30T08:51:46 IST">
      <class name="com.testcases.MakeMyTripTestCase">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.testcases.MakeMyTripTestCase@7a30d1e6]" name="setUp" is-config="true" duration-ms="7431" started-at="2022-07-30T08:51:33 IST" finished-at="2022-07-30T08:51:41 IST">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="logintestfirst()[pri:0, instance:com.testcases.MakeMyTripTestCase@7a30d1e6]" name="logintestfirst" duration-ms="4862" started-at="2022-07-30T08:51:41 IST" finished-at="2022-07-30T08:51:46 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.testcases.MakeMyTripTestCase.logintestfirst(MakeMyTripTestCase.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintestfirst -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.testcases.MakeMyTripTestCase@7a30d1e6]" name="tearDown" is-config="true" duration-ms="108" started-at="2022-07-30T08:51:46 IST" finished-at="2022-07-30T08:51:46 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.testcases.MakeMyTripTestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
